Jenkinsfile-Release

```diff
@@ -71,3 +69,24 @@ dockerizedBuildPipeline(
     buildNotifications(currentBuild, env, 'main')
   }
 )
+
+String verifyIndexYaml(String chart) {
+  // Get current version
+  def indexFile = readYaml file: 'docs/index.yaml'
+  String version = indexFile.entries[chart][0].version
+
+  sleep(60) // seconds
```

Comment:
in another story, i have a reminder that you suggested a `waitFor` method in jenkins groovy for this sort of problem that should be wait and retry. Is this a place you could maybe apply waitFor?

Reply:
Yeah you are right, for some reason I was thinking that could be applied at the library itself, but I am going to confirm that before making any change